src/game_initialization/lobby_data.cpp

```diff
@@ -101,7 +101,7 @@ const std::string& spaced_em_dash()
 std::string make_game_type_marker(const std::string& text, bool color_for_missing)
 {
 	if(color_for_missing) {
-		return markup::bold(markup::span_color("#f00", "[", text, "] "));
+		return markup::span_color("#f00", markup::bold("[", text, "] "));
```

Comment:
So `bold` escapes and `span_color` does not? That seems bound to trip people up again. Also the escaping looks like just a random part of pango markup escaping instead of using `font::escape_text`.

I would suggest to do proper escaping and support a flag to turn it off. All functions should have the same default whether it is escaping or not. Probably best to escape by default since that should be obvious quickly and not only once the right string is injected.

Reply:
Most of pango markup (and probably help markup too) would break if I turn on escaping by default. I was thinking about removing escaping from markup namespace and instead do it explicitly wherever it is needed. (I don't think many need it, not sure.)

Reply:
If it is made very clearly that the default is injection of markup so that every involved variable/input parameter is expected to already be escaped or contain valid markup then that's fine. Currently not even the documentation of these functions gives a hint what is expected input.

I can certainly imagine that the majority of uses will intentionally inject input that is already expected to be valid markup but the expectation of anyone using these functions is unlikely to be so. If the default should be no escaping then a function with a name that gives no hint towards that is just waiting to be misused IMO.